src/core/server/rendering/rendering_service.tsx

```diff
@@ -40,21 +52,32 @@ export class RenderingService {
           mode: this.coreContext.env.mode,
           packageInfo: this.coreContext.env.packageInfo,
         };
+        const buildNum = env.packageInfo.buildNum;
         const basePath = http.basePath.get(request);
         const { serverBasePath, publicBaseUrl } = http.basePath;
         const settings = {
           defaults: uiSettings.getRegistered(),
           user: includeUserSettings ? await uiSettings.getUserProvided() : {},
         };
+
+        const darkMode = getSettingValue('theme:darkMode', settings, Boolean);
+        const themeVersion = getSettingValue('theme:version', settings, String);
+
+        const stylesheetPaths = getStylesheetPaths({
+          darkMode,
+          themeVersion,
+          basePath: serverBasePath,
+          buildNum,
+        });
```

Comment:
Moving the stylesheets loading from `bootstrap.js` to the core/app document introduced a slight behavior change:

- In `bootstrap.js`, we were checking if we were allowed to query ES to retrieve the user settings (security off OR security ON + user authenticated)
- In the rendering service, we depends on the `includeUserSettings` input parameter, which is always `false` for unauthenticated routes (such as `login`)

In short: previously, an authenticated user accessing the `login` page would be able to use the persisted user settings, and now, the default setting values will always be used. As we are reloading the app when logging in, I think this is okay.

If not, we will have to re-use the `isAuthenticated` method/logic from `rendering/bootstrap/bootstrap_renderer.ts` instead of this `includeUserSettings` parameter (which could then  be totally removed from the `render` API)

No strong opinion on that one, tell me what you think.